src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1557,7 +1577,7 @@ private async ValueTask NotifyFeaturesAboutValueChangeAsync(long index, string t
             var taken = false;
             while (taken == false)
             {
-                taken = await _clusterLocker.WaitAsync(TimeSpan.FromSeconds(5));
+                taken = await _updateValuesLocker.WaitAsync(TimeSpan.FromSeconds(5));
```

Comment:
object disposed ex can be thrown here?

Reply:
This is handled in the same place
https://github.com/ravendb/ravendb/blob/2955874a713a26a8a57b787c17bdd1be0a4cb4dc/src/Raven.Server/Documents/DatabasesLandlord.cs#L206